go.mod

```diff
@@ -1,10 +1,11 @@
 module github.com/hashicorp/packer-plugin-vmware
 
-go 1.21.0
+go 1.21.13
```

Comment:
Is there a reason we bump this to 1.21.13? Given we're adding some github action here, this seems out-of-scope?

Reply:
Required 1.N.P syntax. See https://github.com/github/codeql/issues/15647#issuecomment-2076962439